docs/user-guide/cli-zftpplugin.md

```diff
@@ -0,0 +1,60 @@
+# IBM® z/OS FTP Plug-in for Zowe CLI
+
+The IBM® z/OS FTP Plug-in for Zowe&trade; CLI lets you extend Zowe CLI to access z/OS datasets, USS files, and submit JCL. The plug-in uses the z/OS FTP service to achieve the interaction with z/OS.
+
+  - [Use Cases](#use-cases)
+  - [Commands](#commands)
+  - [Software requirements](#software-requirements)
+  - [Installing](#installing)
+  - [Creating a user profile](#creating-a-user-profile)
+
+## Use cases
+
+As a z/OS user, you can use the plug-in to perform the following tasks:
+
+  - List, view, rename, and download z/OS datasets or USS files.
+  - Upload local files or `stdin` to z/OS datasets or USS files.
+  - List, view, and download job status or job spool files.
+  - Delete a z/OS dataset, USS file, or job.
+
+## Commands
+
+For detailed documentation on commands, actions, and options available in this plug-in, see our Web Help. It is available for download in three formats: a PDF document, an interactive online version, and a ZIP file containing the HTML for the online version.
+
+- <a href="../web_help/index.html" target="_blank">Browse Online</a>
+- <a href="../zowe_web_help.zip">Download (ZIP)</a>
+- <a href="../CLIReference_Zowe.pdf">Download (PDF)</a>
+
+## Software requirements
+
+Before you install the plug-in, meet the software requirements in [Software requirements for Zowe CLI plug-ins](cli-swreqplugins.md).
+
+## Installing
+
+Use one of the following methods to install or update the plug-in:
+
+- [Installing plug-ins from an online registry](cli-installplugins.md#installing-plug-ins-from-an-online-registry)
+
+- [Installing plug-ins from a local package](cli-installplugins.md#installing-plug-ins-from-a-local-package)
+
+## Create a User Profile
+
+You can create a `zftp` user profile to avoid typing your connection details on every command. A `zftp` profile contains the host, port, username, and password for the z/OS instance to which you want to connect. You can create multiple profiles and switch between them as needed.
+
+Issue the following command:
+
+    ```
+    zowe profiles create zftp <profile name> -H <host> -u <user> -p <password> -P <port>
+    zowe profiles create zftp <profile name> -H <host> -u <user> -p <password> -P <port> --secure-ftp true
+    ```
```

Comment:
This trick doesn't seem to work in this case, because removing the ticks causes the indented paragraph below to _also_ become part of the code block. Maybe you can show me more about this one day soon.